Dockerfile.ci

```diff
@@ -1,12 +1,29 @@
 FROM dependabot/dependabot-core
-ARG CODE_DIR=/home/dependabot/dependabot-core
+
+USER root
+
+ARG DEPENDABOT_HOME=/home/dependabot
+ARG USERNAME=dependabot
+
+# Install project dependencies as an unprivileged user
+RUN useradd -m ${USERNAME} && mkdir ${DEPENDABOT_HOME}/dependabot-core
+RUN chown -R ${USERNAME}:${USERNAME} \
+  ${DEPENDABOT_HOME}/dependabot-core \
+  /usr/local/.pyenv \
+  /opt/go/gopath \
+  /opt/rust/ \
+  /opt/bundler/
+
+ARG CODE_DIR=${DEPENDABOT_HOME}/dependabot-core
 WORKDIR ${CODE_DIR}
 
-ENV BUNDLE_PATH="/home/dependabot/.bundle" \
-  BUNDLE_BIN=".bundle/binstubs" \
-  PATH=".bundle/binstubs:$PATH:/home/dependabot/.bundle/bin"
+USER ${USERNAME}
+
+ENV BUNDLE_PATH="${CODE_DIR}/.bundle" \
+  BUNDLE_BIN=".bundle/bin"
+ENV PATH="$BUNDLE_BIN:$PATH:$BUNDLE_PATH/bin"
```

Comment:
`$BUNDLE_BIN` and `$BUNDLE_PATH/bin` are now the same value of `.bundle/bin` - do we still need both?

Reply:
Yeah this tripped me up several times, the `BUNDLE_BIN` is set to a relative path but the `BUNDLE_PATH` isn't

Reply:
🦅 👁️ 

Good catch!